docs/manuals/core/config/scenario-config.md

```diff
@@ -64,3 +64,15 @@ associated to a Cycle corresponding to their creation date. See documentation on
 [Scenario and cycle management](../entities/scenario-cycle-mgt.md).
 
 [:material-arrow-right: The next section introduces the global configuration](global-config.md).
+
+# Scenario configuration with comparators
+
+Here is a simple example. Assuming the pipeline configuration `pipeline_config` has already been created with `datanode_config` as one of the data node config in the pipeline config. The `compare_method` function is also predefined:
```

Comment:
Let us imagine a common situation where the pipeline configuration `pipeline_config` has been created with `datanode_config` as one of the data node configuration for this the pipeline configuration.
The function `compare_function` has also been defined: